### DISCUSSION TITLE
Custom reporter and boolean blocks

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#541] [Ideas]
Is it possible to add the ability to let people make their own Operator and Boolean blocks like in [PenguinMod](https://studio.penguinmod.site/editor.html)?

![Screenshot 2023-06-07 at 10 58 42 AM](https://github.com/TurboWarp/extensions/assets/135789443/9a5ca799-2c85-45fc-b7e7-a03c5893d97b)

**Comment by Sunny-da-moth** [comment]
im currently trying to make something sorta like that but on a smaller scale: a reporter block with an id that waits until command block with its id runs and reports the additional value the command block had, when its working i can make the report blocks terminal and make a hat block that runs when reporters with their id start waiting

**Reply by LilyMakesThings** [reply]
https://experiments.turbowarp.org/return/editor
